rotkehlchen/db/evmtx.py

```diff
@@ -594,3 +581,27 @@ def get_or_create_genesis_transaction(
                     relevant_address=account,
                 )
         return tx
+
+    def _get_evm_transaction_query(self, query: str, has_premium: bool) -> str:
+        """Return queries for the evm_transaction database table"""
+        if has_premium:
+            return 'SELECT DISTINCT evm_transactions.tx_hash, evm_transactions.chain_id, timestamp, block_number, from_address, to_address, value, gas, gas_price, gas_used, input_data, nonce FROM evm_transactions ' + query  # noqa: E501
+        else:
+            return 'SELECT DISTINCT evm_transactions.tx_hash, evm_transactions.chain_id, timestamp, block_number, from_address, to_address, value, gas, gas_price, gas_used, input_data, nonce FROM (SELECT * from evm_transactions ORDER BY timestamp DESC LIMIT ?) AS evm_transactions ' + query  # noqa: E501
+
+    def _build_evm_transaction(self, result: Any) -> EvmTransaction:
```

Comment:
Do not use the `Any` type, it's effectively no type.

```suggestion
    def _build_evm_transaction(self, result: tuple[Union[str, int]]) -> EvmTransaction:
```